build_library/grub.cfg

```diff
@@ -37,11 +37,11 @@ if [ "$secure_boot" = "0" ]; then
    fi
 fi
 
-# Determine if this is a first boot. Run Ignition if so.
+# Determine if this is a first boot.
 search --no-floppy --set first_boot \
        --disk-uuid 00000000-0000-0000-0000-000000000001
 if [ -n "$first_boot" ]; then
-    set ignition="coreos.ignition.run=1"
+    set coreos="coreos.first_boot=1"
```

Comment:
Would it be useful to pass along the first-boot disk?

`set coreos="coreos.first_boot=$first_boot"`